__tests__/date.test.ts

```diff
@@ -51,4 +51,77 @@ describe("date", () => {
 
     expect(actual).toBe(expected)
   })
+
+  it("parseDate with not a date string", () => {
+    const dateParser = () => parseDate("not date string")
+    const expected = 'Cannot parse passed date: "not date string"'
+    expect(dateParser).toThrow(expected)
+  })
+
+  it("parseDate with a date string", () => {
```

Comment:
```suggestion
  it("parseDate with a valid date string", () => {
```

Reply:
Apply similar naming to the others named similarly

Reply:
addressed